src/main/resources/application.properties

```diff
@@ -13,7 +13,7 @@
 #
 
 spring.cloud.refresh.refreshable=none
-server.port=8080
+server.port=7750
```

Comment:
You are using 8080 below.

I am fine with keeping 7750 here.
I will send a PR with docs about running Pulsar Manager without NGinx.

Reply:
Thank you very much. I think if we want to modify the default port number, you should modify it together with the front end and the back end, and update the related documents together, otherwise, it will cause confusion for users